plonk-core/src/lookup/lookup_table.rs

```diff
@@ -34,7 +34,7 @@ where
     }
 
     /// Returns the length of the `LookupTable` vector.
-    fn len(&self) -> usize {
+    fn _len(&self) -> usize {
```

Comment:
The CI was complaining about it being unused. I didn't see an obvious place where it should actually *be* used but didn't want to remove it, in case I missed something, so I gave it the underscore tag.